### PR TITLE
new locale: ca_ES

### PR DESCRIPTION
**Pull request by mtub**
files provided by @marcbria

pkp/pkp-lib#1225

### CODE REVIEW

File: locale/ca_ES/default.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../dtd/locale.dtd">
+
+<!--
+  * locale/ca_ES/default.xml
+  *
+  * Copyright (c) 2014-2015 Simon Fraser University Library
```
Comment (asmecher) [inline locale/ca_ES/default.xml]: Copyright dates need updating.

Comment (mtub) [inline locale/ca_ES/default.xml]: Everything is set to the copyright dates in the corresponding en_US locale files. Couldn't we just update all copyright dates in all files after merging these changes?

Comment (asmecher) [inline locale/ca_ES/default.xml]: I already did batch changes for the branches we're likely to release. I suspect we won't release anything further from the omp-dev-1_1 branch, and yes, I'll correct the dates there before releasing if we do.

**Comment by mtub** [comment]
@asmecher, this should be good then…

**Comment by asmecher** [comment]
Phew! Thanks, @mtub.

**Comment by marcbria** [comment]
Thanks you both!! ;-)

Alec, this translation could fail with a very few missing chains because it's before the very last version, but we plan to will check it very soon (for OCS and OMP) with the translation module.
But IMHO you don't need to worry is we won't be able to arrive for this release because the translations are perfectly usables.

**Pull request merged**